core/src/main/scala/cats/effect/test/TestConsole.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect.test
+
+import cats.data.Chain
+import cats.effect.concurrent.Ref
+import cats.effect.{ Console, Sync }
+import cats.syntax.functor._
+import cats.syntax.show._
+import cats.{ Applicative, Show }
+
+private class TestConsole[F[_]: Applicative](outLines: Ref[F, Chain[String]],
```

Comment:
This is me being very picky :p If you put an enter after the opening parens and another enter after the closing parens it'll be formatted like this instead:

```scala
private class TestConsole[F[_]: Applicative](
    outLines: Ref[F, Chain[String]],
    outWords: Ref[F, Chain[String]],
    outErrors: Ref[F, Chain[String]],
    val readLn: F[String]
) extends Console[F] {
```

I tend to prefer this style and I couldn't find a way to force it with `scalafmt`, let me know if you find the way :smile: